c-example/example.c

```diff
@@ -0,0 +1,50 @@
+#include <stdio.h>
+
+#include "spglib.h"
+
+int main(void) {
```

Comment:
Title comment here or above headers. Also short introduction in current folder Readme.

Reply:
Nitpick:

```suggestion
int main(int argc, char* argv[]) {
```

Good to enforce standard practices even in examples.

Reply:
Yes. Done at 4821e31..

Reply:
You can just mark the conversation resolved for these small changes